project/plugins.sbt

```diff
@@ -20,3 +20,7 @@ buildInfoKeys := Seq[BuildInfoKey](buildClasspath)
 buildInfoPackage := "scalabuild"
 
 libraryDependencies += "com.typesafe" %% "mima-reporter" % "0.1.13"
+
+concurrentRestrictions in Global := Seq(
```

Comment:
6 of 1, half dozen of the other:

https://github.com/sbt/sbt/blob/v0.13.13/main/src/main/scala/sbt/EvaluateTask.scala#L206-L219